scout/server/blueprints/institutes/templates/overview/utils.html

```diff
@@ -123,8 +124,32 @@ <h5 class="header">General Institute Settings</h5>
       </div>
     </div>
   </div> <!-- end of div class="row" -->
-</div> <!-- end of container div -->
+
+</form>
+
+<!-- Display form to add a Beacon dataset if user is an andmin and if institute lacks one or more Beaocn datasets -->
+{% if current_user.is_admin and config.BEACON_URL and config.BEACON_TOKEN %}
+<form action="{{ url_for('overview.add_beacon_dataset', institute_id=institute._id) }}" class="form-inline" method="post">
```

Comment:
```suggestion
<form action="{{ url_for('overview.add_beacon_dataset', institute_id=institute._id) }}" method="post">
```
Or `class="row-cols-*"` but I dont think you need it.

Reply:
Ah there was a form-inline. I think you meant that there was no need because I didn't have a form-inline before, that's why I've marked it as resolved, sorry!

Reply:
Yes, no need for it, thanks!

<img width="1106" alt="image" src="https://user-images.githubusercontent.com/28093618/171133608-0caa55c0-3bc8-417b-8333-d15a8e05f010.png">